frontend/src/actions/email_enroll_page.js

```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+
+const getEmail = () => {
+  return dispatch => {
+    const token =
+      "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiaXNBZG1pbiI6dHJ1ZSwiaWF0IjoxNTgxMzgzNDc3LCJleHAiOjE1ODM5NzU0Nzd9.fLn5jTbyPzUMTN-h61DUQtgEdzAXUZMczGqkzFOuwT8";
```

Comment:
I think we should extract this to a function that can be called from all the different places. as this token is used all over the place. 

OR see about using https://www.npmjs.com/package/dotenv to inject the token